src/features/tests/APIWithHooks.spec.ts

```diff
@@ -28,6 +29,7 @@ describe('APIWithHooks', () => {
         postForm: 'postForm',
         delete: 'delete',
         put: 'put',
+        patch: 'PATCH',
```

Comment:
can we use patch instead of PATH to keep all value in lowercase ?

Reply:
I used 'patch' yesterday, but it was refused by CORS. I can try again.